Changes Emag price to 6tc, Removes Budget Emag from Uplink

SgtHunk opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Lowers Normal Emag's price to 6tc, and Removes Budget Emag from Uplink.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

The only reason Emag was so expensive was because of people that would use it to spam open airlocks and cheese it. Now that it no longer can, the price should be changed back to how it was. (TG has it on 4TC, and yes, I do know that this is similar to Drack's Emag tweak PR that got lost in between the stale closed PRs.)

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: tweaked Emag's price, as it's price change was mostly due to being able to spam open airlocks (Which it can't now)
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

SgtHunk committed:
Expensive Emag no more

Lowers Normal Emag's price to 4tc, and Budget Emag's to 2TC. States on Budget that it can't open airlocks.

E231kei500bandai commented:
2 TC is way too cheap. Just delete budget emag and change normal emag to 6 TC.

Helianthus00 commented:
I agree. I think we should return to the tg emag/doormag. This is such a small change anyway that it isn't worth maintaining.

SgtHunk committed:
Removes Budget Emag from Uplink, tweaks Emag price once again

Removes Budget from Uplink (Still in code), changes normal Emag's price back to 6tc (Yes, TG has it at 4 but I feel like 6 fits a bit more.)

BarrelOfNapkins commented:
If TG has it at 4TC why not just make it 4TC? 4 vs 6 tc seems like tomato potato difference, not worth it for incoming merge conflicts.

SgtHunk commented:
6tc feels alright considering that it still can fuck up most things even if its airlock feature got (mostly) removed
If Sam wants me to, I can simply just tweak it back to 4tc and basically revert all of those changes

SgtHunk committed:
Sometimes my genius is... it's almost frightening

Changes this back because Surreal's overwrite will do the job. Also saves the headache of a merge conflict
Remember kids: This is what happens when you try to code at 1 AM with 4 hours of sleep and surviving purely off of coffee

SgtHunk committed:
Fulfills Brazil Man's request

Adds a traitor_overwrite files to make things easier, requested by Guill so might as well.

EgorDinamit commented:
Emag is not really useful for most traitors on Fulp. Just revert it back to 4 TC.

E231kei500bandai commented:
I see tons of people buying it at 15 TC, idk why they buy it cause the budget emag is a way better deal but they buy it at 15 tc
